.github/workflows/nodejs.yml

```diff
@@ -36,5 +39,17 @@ jobs:
       run: npm i
 
     - name: Test
-      run: npm test
+      id: test
+      run: |
+        npm run test-coverage-ci || echo "exit_code=$?" >> $GITHUB_OUTPUT
+
+    - name: Comment test coverage report on PR
+      uses: romeovs/lcov-reporter-action@v0.2.16
+      with:
+        lcov-file: ./coverage/lcov.info
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Exit if coverage condition not met
+      if: ${{ steps.test.outputs.exit_code }} != 0
```

Comment:
@JamieSlome until we close #240, I would err on the side of removing this explicit failure in the workflow (maybe commenting out for now to be re-introduced after coverage is brought up to the thresholds?). 

For the time being, coverage can be reported in PRs for informational purposes.

Reply:
@coopernetes, agreed 👍 No use blocking PRs and test creation should be incremental in any case.

Reply:
Agreed.
I'll comment out the explicit failure part.